docs/datamodel/scalars/numeric.rst

```diff
@@ -77,6 +77,36 @@ from :eql:type:`str` and :eql:type:`json`.
 ----------
 
 
+.. eql:type:: std::bigint
+
+    :index: numeric bigint
+
+    Arbitrary precision integer.
+
+    The EdgeDB philosophy is that using decimal type should be an
```

Comment:
This should be updated for bigint? And the next paragraph?

Reply:
Done